recipes/amazon-sagemaker-sql-execution/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "amazon-sagemaker-sql-execution" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/amazon-sagemaker-sql-execution-{{ version }}.tar.gz
+  sha256: 08b8e6be09f6ecaad326ec3dc5c581d4497a57b986ca0aafcf779250bec3da80
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools
+    - pip
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - amazon_sagemaker_sql_execution
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/amazon-sagemaker-sql-execution/
+  summary: SageMaker SQL Magic library
```

Comment:
minor comment - can you double check the summary? (doesn't match with library name)

Reply:
Updated with improved summaries.